pkg/file/io.go

```diff
@@ -107,3 +107,18 @@ func SplitWriteAll(ctx context.Context, s Splitter, r io.Reader, l int64, toEncr
 	}
 	return addr, nil
 }
+
```

Comment:
these interfaces are defined twice. we should use only one and have it in `storage` package maybe

Reply:
these interfaces provide functionalities which are wrapping components in the `file` package. that's why I put them here. it belongs here more than in `storage`, which relates more to `localstore` and of such